bot/seasons/easter/april_fools_vids.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+
+class AprilFoolVideos(commands.Cog):
+    """A cog for april fools that gets a random april fools video from youtube."""
+    def __init__(self, bot):
+        self.bot = bot
+        self.yt_vids = self.load_json()
+        self.youtubers = ['google']  # will add more in future
+
+    @staticmethod
+    def load_json():
+        """A function to load json data."""
+        p = Path('bot/resources/april_fools_vids.json')
+        with p.open() as json_file:
+            all_vids = load(json_file)
+        return all_vids
+
+    @commands.command(name='fool')
+    async def aprial_fools(self, ctx):
+        """Gets a random april fools video from youtube."""
+        random_youtuber = 'google'
+
+        # Change the above line of code to "random_youtuber = random.choice(self.youtubers)".
+        # I will add more youtubers and their vids in the future, for now only google, Around 30 vids.
+
+        category = self.yt_vids[random_youtuber]
```

Comment:
lines 32 `random_youtuber = 'google'`, 37 `category = self.yt_vids[random_youtuber]` and 19 serves towards one end. They can be combined.

Reply:
Well incase we add more youtubers , this line will help? or should i just i do wut u said and change stuff when we have more youtubers?

Reply:
If you want to leave the selection open to future expansion then randomly select from `self.youtubers` as you've described in the code comment rather than hardcoding it to `'google'` in the command itself.

Reply:
okay